configs/components/yaml-cpp.rb

```diff
@@ -53,6 +53,7 @@
   elsif platform.is_solaris?
     # We always use the i386 build of cmake, even on sparc
     cmake = "#{settings[:basedir]}/i386-pc-solaris2.#{platform.os_version}/bin/cmake"
+    pkg.environment "PATH" => "$$PATH:/opt/csw/bin"
```

Comment:
Makefiles require double-escaping any dollar sign that should be interpreted by the shell